projects/layout/components/card/collapsed.style.less

```
@@ -0,0 +1,44 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiCardCollapsed] {
+    .transition(~'clip-path, margin');
+
+    border-image: radial-gradient(at center, var(--tui-background-neutral-2) 26.5%, transparent 27%) 50% 49% 50% / 0
```

Do we need the bottom-value if it is equal to top one ?

```suggestion
    border-image: radial-gradient(at center, var(--tui-background-neutral-2) 26.5%, transparent 27%) 50% 49% / 0
```